internal/pkg/convert/graph.go

```diff
@@ -55,6 +61,23 @@ func NewGraphLLB(graph *solver.PackageGraph, options *environment.Options) *Grap
 	return result
 }
 
+func (graph *GraphLLB) convertPlatform(platform string) *v1.Platform {
```

Comment:
this seems to be not related to the graph, probably move it to be a function on its own (to a separate file)?